.travis.yml

```diff
@@ -2,29 +2,33 @@ language: php
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION="4.0.x-dev"
+    - COMPOSER_ROOT_VERSION="2.0.x-dev"
 
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
+      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1 SUBSITES=1
     - php: 7.0
-      env: DB=MYSQL PHPUNIT_TEST=1
+      env: DB=PGSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1 SUBSITES=1
+    - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1
 
 before_script:
   - phpenv rehash
   - phpenv config-rm xdebug.ini
 
-  - composer install --prefer-dist
-  - composer require --prefer-dist --no-update silverstripe/recipe-cms:1.0.x-dev
-  - composer update
+    - composer validate
+    - composer require --no-update silverstripe/recipe-cms:1.0.x-dev
+    - if [[ $SUBSITES ]]; then composer require --no-update silverstripe/subsites:2.0.x-dev; fi
+    - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql:2.0.x-dev; fi
+    - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
```

Comment:
This indentation is whack, you'll need to fix it up (to be inline with line 19 and 20) before Travis will pass